node/v6/index.d.ts

```diff
@@ -347,6 +347,14 @@ declare namespace NodeJS {
         openssl: string;
     }
 
+    type Platform = 'aix'
+                  | 'darwin'
+                  | 'freebsd'
+                  | 'linux'
+                  | 'openbsd'
+                  | 'sunos'
+                  | 'win32';
```

Comment:
I didn't know that Node.js supports Android experimentally. Thank you for catching it.